doc/website/blog/2023-12-12-hibernate-schema-loader.mdx

```diff
@@ -0,0 +1,598 @@
+---
+title: "Announcing Automatic Migrations for Hibernate Users"
+authors: dorav
+tags: [hibernate, schema, migration]
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+:::info TL;DR
+
+You can now import the desired database schema from your Hibernate project into Atlas,
+and use it to automatically plan migrations for you.
+
+[See an example](#demo-time)
+
+:::
+
+## Introduction
+
+[Hibernate-ORM](https://hibernate.org/orm/documentation/), is one of the most popular ORMs for Java, so much so that parts of it have evolved into
```

Comment:
```suggestion
[Hibernate-ORM](https://hibernate.org/orm/documentation/) is one of the most popular ORMs for Java, so much so that parts of it have evolved into
```